README.md

```diff
@@ -45,7 +45,7 @@ If you want to run the aspect model editor from repositories, please ensure to c
 # enter the core directory where the package.json is located
 cd core
 
-npm install
+npm install --legacy-peer-deps
```

Comment:
Using --legacy-peer-deps is a quick fix for dependency conflicts, but adjusting the package.json is a more sustainable and stable solution to prevent long-term incompatibilities.

Thank you for your contribution! Please revert your changes and update the Nx version in [package.json](https://github.com/eclipse-esmf/esmf-aspect-model-editor/blob/5c810e662446cdcf606695518e14bb1105ee1776/core/package.json#L301) to 19.1.0, then commit the update. This should fix the problem.